foundations/javascript_basics/revisiting_rock_paper_scissors.md

```diff
@@ -35,6 +44,7 @@ gitGraph
    commit id: "commit2a"
    checkout main
    merge develop id: "merge to main"
+   
```

Comment:
```suggestion

```
Removing these spaces so the line is truly blank should fix it (and is exactly why we now need the blank lines thing, because not having them causes the linter to not read some lines properly, so it misses some errors until the blank lines are inserted.